plotly/plotly/plotly.py

```diff
@@ -1209,14 +1275,20 @@ def parse_grid_id_args(cls, grid, grid_url):
 
     @classmethod
     def response_handler(cls, response):
-
         try:
             response.raise_for_status()
         except requests.exceptions.HTTPError as requests_exception:
-            plotly_exception = exceptions.PlotlyRequestError(
-                requests_exception
-            )
-            raise(plotly_exception)
+            if (response.status_code == 404 and
+                    get_config()['plotly_api_domain']
```

Comment:
is `get_config()[...]` for sure what was used to make this request? Could it have been something different if they `py.sign_in`'d, e.g. something from `session`? (I'm not actually sure, I just thought `get_config` always grabs the config from file)